pkg/deploy/stack_deployer.go

```diff
@@ -235,3 +234,29 @@ func (d *latticeTargetsStackDeployer) Deploy(ctx context.Context, stack core.Sta
 	}
 	return nil
 }
+
+type accessLogSubscriptionStackDeployer struct {
+	log                          gwlog.Logger
+	k8sClient                    client.Client
+	stack                        core.Stack
+	accessLogSubscriptionManager lattice.AccessLogSubscriptionManager
```

Comment:
As long as there is single manager name can be simplified to manager, I dont see there will be multiple managers here.

Reply:
Fair enough, fixed next commit